src/elements/content-sidebar/activity-feed/annotations/messages.ts

```diff
@@ -0,0 +1,52 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+    annotationActivityDeleteMenuItem: {
+        id: 'be.contentSidebar.activityFeed.annotationActivity.annotationActivityDeleteMenuItem',
```

Comment:
I see some of these ids have `annotationActivity` in the path and some don't. If it's in the same file they all should have the same path right? I've seen we generally follow the filepath so wouldn't the path be `be.contentSidebar.activityFeed.annotations`

Reply:
It is basically copied content from `messages.flow.js` to be able to use it in typescript component (I've found this pattern in `src/components/badge` - let me know if there is a better way to handle translation messages in flow and ts at the same time). I would rather not change ids of translations which are already in the codebase for some time.